ts/util/arrayBufferToObjectURL.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @prettier
+ */
+import { MIMEType } from '../types/MIME';
+
+export const arrayBufferToObjectURL = ({
```

Comment:
Very important little function, for sure. I had reason to write this a few times myself - for the styleguide, for handling attachments in quoted messages...

One thing to think about - is the right code in place to revoke these URLs once created?

Reply:
Indeed. I think revocation should lie with the caller as it depends on the context. Once I have infinite scrolling, I can look into that.